ts/components/ui/IOScrollViewWithLargeHeader.tsx

```diff
@@ -111,10 +115,15 @@ export const IOScrollViewWithLargeHeader = forwardRef<View, Props>(
         {description && (
           <ContentWrapper>
             <VSpacer size={4} />
-            <Body color="grey-700">{description}</Body>
+            {typeof description === "string" ? (
+              <Body color="grey-700">{description}</Body>
+            ) : (
+              <ComposedBodyFromArray body={description} textAlign="left" />
+              // This left alignement is the default, it can be overwritten
```

Comment:
The title is always left-aligned, so I can't imagine a screen that would need a different text alignment. Can we disable override alignment altogether in this particular case?

Reply:
No. The alignment can be customised with the style rules that accept the body as props.